tests/api/test_product_minimal_variant_price.py

```diff
@@ -3,117 +3,12 @@
 import graphene
 from freezegun import freeze_time
 from graphql_relay import from_global_id, to_global_id
-from prices import Money
 
 from saleor.graphql.discount.enums import DiscountValueTypeEnum
 from saleor.product.error_codes import ProductErrorCode
 from tests.api.utils import get_graphql_content
 
 
-def test_product_create_sets_minimal_variant_price(
```

Comment:
We don't need this test because it is based on a removed field (`price`) to update `minimal_variant_price`, since this field is removed, this test case is invalid.

Reply:
So what values I will get for field `minimalVariantProduct` when I call query:
```
{
  products(first:35, filter:{price: { lte: 20}}){
    totalCount
    edges{
      node{
        id
        minimalVariantPrice{
          amount
        }
      }
    }
  }
}
```
Assuming that I have mixed Product With and Without Variants.

Reply:
All products have variants (or at least all products that you can purchase have them).